tests/relay_integration/lang/java/test_plugin.py

```diff
@@ -497,6 +505,12 @@ def test_basic_resolving(self):
             "org.slf4j.helpers.Util$ClassContextSecurityManager " "in getExtraClassContext"
         )
 
+    @requires_symbolicator
+    @pytest.mark.symbolicator
+    def test_basic_resolving_symbolicator(self):
+        with override_options({"symbolicator.proguard-processing-sample-rate": 1.0}):
+            self.test_basic_resolving()
```

Comment:
IMO its okay to duplicate all these tests. Though ideally you can dig up the code we had previously for the JS rollout to properly parameterize the existing tests.

Reply:
That unfortunately doesn't work—it used a fixture for parametrization and these tests don't support fixtures (something about `unittest` vs `pytest`).